### PR TITLE
Updates concerning the new Monitor app

### PR DESCRIPTION
**Pull request by marc-tonsen**

### CODE REVIEW

File: src/invisible/getting-started/understand-the-ecosystem.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Note that without uploading to Pupil Cloud, the 200 Hz gaze signal is not availa
 
 All data generated by Pupil Invisible can be monitored in real-time using the Pupil Invisible Monitor app. To access the app simply visit `pi.local` in your browser while being connected to the same WiFi network as the Companion device.
 
-It allows you to stream scene video and gaze data as well as to remotely control the Companion app to start/stop recordings and save events. This way you can fully control and monitor an ongoing recording session without having to directly interact with the Companion device carried by the subject. -->
+It allows you to stream scene video and gaze data as well as to remotely control the Companion app to start/stop recordings and save events. This way you can fully control and monitor an ongoing recording session without having to directly interact with the Companion device carried by the subject.
```
Comment (willpatera) [inline src/invisible/getting-started/understand-the-ecosystem.md:61]:
```suggestion
The real-time streaming app enables you to monitor and control a recording session remotely without having to directly interact with the Companion device carried by the subject. You can view the scene video and gaze data remotely in your browser. You can also start and stop recordings and annotate important moments in time with events. All data will be saved with the recording on the Companion device.
```

Comment (willpatera) [reply src/invisible/getting-started/understand-the-ecosystem.md:61]: Link to events?

File: src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md [hunk 2]
```diff
@@ -4,14 +4,38 @@ description: Links to Pupil Invisible Monitor app.
...
   >
   </v-img>
 </div>
 
-**Coming soon!** We are finishing up work on a new version of Pupil Invisible Monitor, which will allow you to easily monitor and remote-control all Pupil Invisible devices in your local network from your browser.
+## Connection Problems?
+
+In case you have trouble connecting to your Pupil Invisible device please consider the following:
+- First, make sure the Companion phone and the device you are using for monitoring are both connected to the same network.
+- The the local network must allow MDNS and UDP traffic for the app to work. In large public networks this may be prohibited for security reasons. You can circumvent this by running a separate WiFi using the phone's hot spot functionality or an extra router.
```
Comment (willpatera) [inline src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md:31]: Links to MDNS and UDP?

Comment (willpatera) [inline src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md:31]: Internet connectivity not required.

File: src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md [hunk 3]
```diff
@@ -4,14 +4,38 @@ description: Links to Pupil Invisible Monitor app.
...
+
+If you still have trouble, please reach out to us in [chat](https://pupil-labs.com/chat/)!
+
+## Conclusion
+
+The Pupil Invisible Monitor app allows you to easily monitor your data collection and remotely control all your Pupil Invisible devices.
```
Comment (willpatera) [inline src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md:37]: Do we need this part?

**Review by willpatera — CHANGES_REQUESTED** [review]
Looks good - I think though we can improve the getting started content.

File: src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md (hunk 2 above)
Comment (marc-tonsen) [reply src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md:31]: Where would those point to? Wikipedia? Not sure if that would add much value but, wouldn't hurt either!

File: src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md (hunk 3 above)
Comment (marc-tonsen) [reply src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md:37]: Unsure. Felt weird to me to start directly with the first reference without a summary sentence first, but I'd be happy to have this removed too!

File: src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md (hunk 2 above)
Comment (mkassner) [reply src/invisible/how-tos/data-collection-with-the-companion-app/monitor-your-data-collection-in-real-time.md:31]: I think its ok without. The solution is listed in the text. Knowing more about MDNS will not help the avg. user.

**Pull request merged**